src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -38,7 +38,7 @@ public GrpcHttpRequestData(RpcHttp httpData, FunctionContext functionContext)
 
                 var cookieString = Headers.FirstOrDefault(item => item.Key.Equals("Cookie", StringComparison.OrdinalIgnoreCase)).Value;
 
-                if (cookieString != null && cookieString.Any())
+                if (cookieString != null && cookieString.Any() && !string.IsNullOrWhiteSpace(cookieString.First()))
```

Comment:
Currently if the cookie string is empty, we are still returning one item with empty name and value properties. This change fixes that.